tests/mounts/mounts_test.go

```diff
@@ -47,35 +43,35 @@ func TestSingleMountDeployment(t *testing.T) {
 	// Check that the outputs not empty
 	publicIP := terraform.Output(t, terraformOptions, "public_ip")
 	assert.NotEmpty(t, publicIP)
+	publicIP, err := tests.IPFromCidr(publicIP)
+	assert.NoError(t, err, "error parsing public ip")
 
 	node1Container1IP := terraform.Output(t, terraformOptions, "node1_container1_ip")
 	assert.NotEmpty(t, node1Container1IP)
+	node1Container1IP, err = tests.IPFromCidr(node1Container1IP)
```

Comment:
i checked the tf files i think `node1Container1IP` and `node1Container2IP` are private ips and aren't CIDR so parsing isn't needed here